repo/packages/M/msoms/1/package.json

```diff
@@ -0,0 +1,23 @@
+{
+ "packagingVersion": "3.0",
+ "name": "msoms",
+ "version": "1.0.1",
+ "scm": "git@github.com:Microsoft/OMS-docker.git",
+ "maintainer": "omscontainers@microsoft.com",
+ "description": "Microsoft Operations Management Suite(MS OMS) – Container Solution provides real-time analytics for operational data (Performance, Inventory) and monitoring for Docker Containers.",
```

Comment:
I will certainly add an example in a separate pull request.
This merge is urgent, we have to check in this for one of our customer. 
Please merge this change if it looks good to you.